<ui version="4.0" >
 <class>UiSearchBase</class>
 <widget class="QWidget" name="UiSearchBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>194</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>580</width>
    <height>194</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>580</width>
    <height>194</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Search for</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resource/dms.qrc" >:/picture/32/images/32x32/search_32.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Search</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="labelSearchFor" >
            <property name="minimumSize" >
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>Word:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSearchWord" />
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSearchExactly" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>exact match</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelSearchColumn" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Column:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSearchColumn" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelSearchGroup" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Group:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSearchGroup" />
        </item>
       </layout>
      </item>
      <item row="3" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelSearchUser" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>User:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSearchUser" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnSearch" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/dms.qrc" />
 </resources>
 <connections/>
</ui>
